spec/requests/users_controller_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+RSpec.describe 'UsersController', type: :request do
+  let!(:user) { User.create!(name: 'Test User') }
+
+  describe 'GET /index' do
+    it 'returns a successful response' do
+      get users_path
+      expect(response).to have_http_status(:success)
+    end
+
+    it 'renders the index template' do
+      get users_path
+      expect(response).to render_template(:index)
+    end
+
+    it "includes the user's name in the response body" do
+      get users_path
+      expect(response.body).to include(user.name)
+    end
+  end
+
+  describe 'GET /show' do
+    it 'returns a successful response' do
+      get user_path(user)
+      expect(response).to have_http_status(:success)
+    end
+
+    it 'renders the show template' do
+      get user_path(user)
+      expect(response).to render_template(:show)
+    end
+
```

Comment:
While you did a good job working on the requirements some of your tests are failing and some are pending all tests should pass so it would be nice to correct this. Kindly take a look at the screenshot below:

Screenshot:

![Screenshot 2023-11-26 at 5 48 24 AM](https://github.com/jlberbesi/blog-app/assets/79506105/ca13f793-fb1d-40bf-839b-a95198155c31)

Reply:
@KayLemba Solved

Reply:
@KayLemba Solved

Reply:
@KayLemba Solved